packages/core/src/utils/SchemaRouter.ts

```diff
@@ -60,6 +61,8 @@ type SchemaRouterConfig<Key extends string> = {
    * @see {@link generateStandardId} for the default length.
    */
   idLength?: number;
+  /** The KoaQuotaGuard params applied to `POST /` route. */
+  quotaGuard?: UsageGuardConfig;
```

Comment:
it looks like this usage has no relation with the koaQuotaGuard middleware

Reply:
also not reasonable to put the library to the config

Reply:
yeah, need to update the comment. Initially I was trying to use the `KoaQuotaGuard` middleware, but it seems there's no easy way to apply the middleware conditionally. So I took the quota guard logic from the middleware and apply it inside the POST requests instead

Reply:
Per our offline discussion, updated the code with a `use` prop and allow injecting a middleware into the `SchemaRouter` class